app/pkg/letteravatar/letteravatar.go

```diff
@@ -0,0 +1,142 @@
+package letteravatar
```

Comment:
This looks interesting. So you use the font to generate the letter avatar, so that it supports arabic letters. That's cool - if we used this we'd just need to make sure we can support both arabic and non-arabic. Do you have any thoughts about how best to manage that?

Reply:
I give up and convert the Arabic letters to English,Ex: ب = B , from my very basic experience with php and converting text to image, Arabic language is very challenging